docs/esa/iso/iso.rst

```diff
@@ -119,75 +115,86 @@ observation identifier (TDT) as input. There are different product levels:
+  INFO: Copying file to 80002304.tar... [astroquery.esa.iso.core]
+  INFO: Wrote http://nida.esac.esa.int/nida-sl-tap/data?retrieval_type=OBSERVATION&DATA_RETRIEVAL_ORIGIN=astroquery&tdt=80002304&product_level=DEFAULT_DATA_SET to 80002304.tar [astroquery.esa.iso.core]
+  '80002304.tar'
+  INFO: http://nida.esac.esa.int/nida-sl-tap/data?retrieval_type=OBSERVATION&DATA_RETRIEVAL_ORIGIN=astroquery&tdt=80002450&product_level=DEFAULT_DATA_SET [astroquery.esa.iso.core]
   INFO: Copying file to 80002450.tar... [astroquery.esa.iso.core]
-  INFO: Wrote http://nida.esac.esa.int/nida-sl-tap/data? 
-  retrieval_type=OBSERVATION&tdt=80002450&product_level=DEFAULT_DATA_SET to 80002450.tar [astroquery.esa.iso.core]
-  ......
-  INFO: http://nida.esac.esa.int/nida-sl-tap/data? 
-  retrieval_type=OBSERVATION&tdt=80000104&product_level=DEFAULT_DATA_SET [astroquery.esa.iso.core]
-  INFO: Copying file to 80000104.tar... [astroquery.esa.iso.core]
-  INFO: Wrote http://nida.esac.esa.int/nida-sl-tap/data? 
-  retrieval_type=OBSERVATION&tdt=80000104&product_level=DEFAULT_DATA_SET to 80000104.tar [astroquery.esa.iso.core]
-  
-  
+  INFO: Wrote http://nida.esac.esa.int/nida-sl-tap/data?retrieval_type=OBSERVATION&DATA_RETRIEVAL_ORIGIN=astroquery&tdt=80002450&product_level=DEFAULT_DATA_SET to 80002450.tar [astroquery.esa.iso.core]
+  '80002450.tar'
+  INFO: http://nida.esac.esa.int/nida-sl-tap/data?retrieval_type=OBSERVATION&DATA_RETRIEVAL_ORIGIN=astroquery&tdt=80002247&product_level=DEFAULT_DATA_SET [astroquery.esa.iso.core]
+  INFO: Copying file to 80002247.tar... [astroquery.esa.iso.core]
+  INFO: Wrote http://nida.esac.esa.int/nida-sl-tap/data?retrieval_type=OBSERVATION&DATA_RETRIEVAL_ORIGIN=astroquery&tdt=80002247&product_level=DEFAULT_DATA_SET to 80002247.tar [astroquery.esa.iso.core]
+  '80002247.tar'
+  INFO: http://nida.esac.esa.int/nida-sl-tap/data?retrieval_type=OBSERVATION&DATA_RETRIEVAL_ORIGIN=astroquery&tdt=80002014&product_level=DEFAULT_DATA_SET [astroquery.esa.iso.core]
+  INFO: Copying file to 80002014.tar... [astroquery.esa.iso.core]
+  INFO: Wrote http://nida.esac.esa.int/nida-sl-tap/data?retrieval_type=OBSERVATION&DATA_RETRIEVAL_ORIGIN=astroquery&tdt=80002014&product_level=DEFAULT_DATA_SET to 80002014.tar [astroquery.esa.iso.core]
+  '80002014.tar'
 
 -------------------------------
 2. Getting ISO postcards
 -------------------------------
 
-.. code-block:: python
+.. doctest-remote-data::
 
   >>> from astroquery.esa.iso import ISO
-  >>> ISO.get_postcard('80001538', filename="postcard")
-  
   >>> from IPython.display import Image
-  >>> from IPython.core.display import HTML 
+  >>> from IPython.core.display import HTML
+  >>>
+  >>> ISO.get_postcard('80001538', filename="postcard")   # doctest: +IGNORE_OUTPUT
   >>> Image(filename = "postcard.png", width=400, height=100)
-  
-  Downloading URL http://nida.esac.esa.int/nida-sl-tap/data?&retrieval_type=POSTCARD&tdt=80001538 to 
```

Comment:
I would still keep this output in the line above to show the users that something is expected here, we just don't run the doctest on it for exact match